cmd/release-controller/main.go

```diff
@@ -161,6 +163,8 @@ func main() {
 
 	flagset.StringVar(&opt.AuthenticationMessage, "authentication-message", opt.AuthenticationMessage, "HTML formatted string to display a registry authentication message")
 
+	flagset.StringVar(&opt.Registry, "registry", opt.Registry, "The image registry to use")
```

Comment:
Use for what?  Describe how the flag is used.

Reply:
I updated the message to:

> Specify an alternate image registry to reference.  This option should be used when running the controller on a different cluster.